models/public/nyu-fcrn-tf/nyu-fcrn-tf.md

```diff
@@ -0,0 +1,109 @@
+# midasnet
+
+## Use Case and High-Level Description
+
+NYU-FCRN is a model for monocular depth estimation trained by NYU Depth V2 dataset;
+  as described in the paper [Deeper Depth Prediction with Fully Convolutional Residual Networks](https://arxiv.org/abs/1606.00373) 
+  The model input is a blob that consists of a single image of "1x228x304x3" in RGB order.
+  The model output is an inverse depth map that is defined up to an unknown scale factor. More details can be found in the [following repository](https://github.com/iro-cp/FCRN-DepthPrediction).
+
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Monodepth     |
+| GFLOPs            | 63.5421       |
+| MParams           | 34.5255       |
+| Source framework  | TensorFlow\*  |
+
+## Accuracy
+
+| Metric | Value |
+| ------ | ----- |
+| [rmse](https://en.wikipedia.org/wiki/Root-mean-square_deviation)   | 0.573 |
+| log10  | 0.055 |
+| [rel](https://en.wikipedia.org/wiki/Mean_absolute_percentage_error)    | 0.127 |
```

Comment:
Is `MAPE` a more appropriate name for metric here?

Reply:
in depth estimation task it usually named as rel

Reply:
I rewrite metric definition to nobody confuse

Reply:
I rewrite metric definition to nobody confuse